committing.rst

```diff
@@ -5,212 +5,155 @@ Accepting Pull Requests
-   will exercise more possibilities still (such as different platforms or
-   build options).  But it will at least catch non-build specific,
-   non-platform specific errors, therefore minimizing the chance for
-   breakage.
-
-
-Patch checklist
-'''''''''''''''
-
-You should also :ref:`run patchcheck <patchcheck>` to perform a quick
-sanity check on the changes.
-
-
-Handling Others' Code
----------------------
-
-As a core developer you will occasionally want to commit a patch created by
-someone else. When doing so you will want to make sure of some things.
-
-First, make sure the patch is in a good state. Both :ref:`patch` and
-:ref:`helptriage`
-explain what is to be expected of a patch. Typically patches that get cleared by
-triagers are good to go except maybe lacking ``Misc/ACKS`` and ``Misc/NEWS.d``
-entries (which a core developer should make sure are updated appropriately).
-
-Second, make sure the patch does not break backwards-compatibility without a
-good reason. This means :ref:`running the entire test suite <runtests>` to
-make sure everything still passes. It also means that if semantics do change
-there must be a good reason for the breakage of code the change will cause
-(and it **will** break someone's code). If you are unsure if the breakage
-is worth it, ask on python-dev.
-
-Third, ensure the patch is attributed correctly with the contributor's
-name in ``Misc/ACKS`` if they aren't already there (and didn't add themselves
-in their patch) and by mentioning "Patch by <x>" in the ``Misc/NEWS.d`` entry
-and the check-in message. If the patch has been heavily modified then "Initial
-patch by <x>" is an appropriate alternate wording.  GitHub now supports `multiple
-authors <https://help.github.com/articles/creating-a-commit-with-multiple-authors/>`_
-in a commit. Add ``Co-authored-by: name <name@example.com>`` at the end of the commit message.
-
-If you omit correct attribution in the initial check-in, then update ``ACKS``
-and ``NEWS.d`` in a subsequent check-in (don't worry about trying to fix the
-original check-in message in that case).
-
-Finally, make sure that the submitter of the
-patch has a CLA in place (indicated by an asterisk following their username
-in the `issue tracker`_ or by the "CLA Signed" label on the pull request).
-If the submitter lacks a signed CLA and the patch is non-trivial, direct them
-to the electronic `Contributor Licensing Agreement`_
-to ensure the PSF has the appropriate authorizations in place to relicense
-and redistribute their code.
-
-
-Contributor Licensing Agreements
---------------------------------
+This page is a step-by-step guide for core developers who need to assess,
+merge, and possibly backport a pull request on the main repository.
+
+Assessing a pull request
+------------------------
+
+Before you can accept a pull request, you need to make sure that it is ready
+to enter the public source tree. Ask yourself the following questions:
+
```

Comment:
```suggestion

* **Are there ongoing discussions in b.p.o.?**
   Read the linked b.p.o. issue and check the pull request is ready to merge. If there are ongoing discussions, we need to make a consensus before merging the pull request.
```

Reply:
@methane I added a modified version of this in a commit. Let me know what you think.